doc/api/subjects/behaviorsubject.md

```diff
@@ -1,6 +1,6 @@
 # `Rx.BehaviorSubject` class #
 
-Represents a value that changes over time.  Observers can subscribe to the subject to receive the last (or initial) value and all subsequent notifications.
+Represents a value that changes over time.  Observers can subscribe to the subject to receive the last (or initial) value and all subsequent notifications. If you are looking for BehaviorSubject without initial value use [`Rx.ReplaySubject`](https://github.com/Reactive-Extensions/RxJS/blob/master/doc/api/subjects/replaysubject.md) and [`take`](https://github.com/Reactive-Extensions/RxJS/blob/master/doc/api/core/operators/take.md) the latest value on subscription instead.
```

Comment:
I'm fine with the first part including a reference to the `ReplaySubject`. I don't understand the second part though. Why are you referencing take here?